### PR TITLE
Update FR production parser to use other API

### PR DESCRIPTION
**Pull request by lorrieq**
* RTE API is not meant to be an open API, use opendata.reseaux-energies.fr

* Add special case for hydro calculation

* rows parameter to return all datetimes

proposes a solution for the desire to switch APIs in #1551

### CODE REVIEW

**Comment by corradio** [comment]
Terrific @lorrieq ! Are you able to post the output of the new parser and the old one so we can check that they return the same thing?

**Comment by lorrieq** [comment]
Damn, forgot again!

`{'zoneKey': 'FR', 'source': 'https://opendata.reseaux-energies.fr', 'production': {'oil': 167.0, 'nuclear': 37550.0, 'gas': 2794.0, 'biomass': 781.0, 'coal': 339.0, 'solar': 3664.0, 'wind': 4453.0, 'hydro': 4809.0}, 'storage': {'hydro': -830.0}, 'datetime': datetime.datetime(2018, 8, 13, 14, 30, tzinfo=tzoffset(None, 7200))}
{'zoneKey': 'FR', 'source': 'https://opendata.reseaux-energies.fr', 'production': {'oil': 167.0, 'nuclear': 37551.0, 'gas': 2877.0, 'biomass': 778.0, 'coal': 342.0, 'solar': 3758.0, 'wind': 4489.0, 'hydro': 4807.0}, 'storage': {'hydro': -623.0}, 'datetime': datetime.datetime(2018, 8, 13, 14, 45, tzinfo=tzoffset(None, 7200))}
{'zoneKey': 'FR', 'source': 'https://opendata.reseaux-energies.fr', 'production': {'oil': 168.0, 'nuclear': 37521.0, 'gas': 2831.0, 'biomass': 784.0, 'coal': 346.0, 'solar': 3585.0, 'wind': 4631.0, 'hydro': 4615.0}, 'storage': {'hydro': -547.0}, 'datetime': datetime.datetime(2018, 8, 13, 15, 0, tzinfo=tzoffset(None, 7200))}
{'zoneKey': 'FR', 'source': 'https://opendata.reseaux-energies.fr', 'production': {'oil': 168.0, 'nuclear': 37773.0, 'gas': 2979.0, 'biomass': 784.0, 'coal': 384.0, 'solar': 3534.0, 'wind': 4691.0, 'hydro': 5035.0}, 'storage': {'hydro': -816.0}, 'datetime': datetime.datetime(2018, 8, 13, 15, 15, tzinfo=tzoffset(None, 7200))}
{'zoneKey': 'FR', 'source': 'https://opendata.reseaux-energies.fr', 'production': {'oil': 169.0, 'nuclear': 37966.0, 'gas': 2906.0, 'biomass': 777.0, 'coal': 390.0, 'solar': 3354.0, 'wind': 4622.0, 'hydro': 4867.0}, 'storage': {'hydro': -691.0}, 'datetime': datetime.datetime(2018, 8, 13, 15, 30, tzinfo=tzoffset(None, 7200))}`

**Comment by corradio** [comment]
I looked for the 'nuclear': 37966.0 value as reference, and the data seems to coincide:

With old parser:
{'zoneKey': 'FR', 'datetime': datetime.datetime(2018, 8, 13, 15, 30, tzinfo=tzfile('/usr/share/zoneinfo/Europe/Paris')), 'production': {'nuclear': 37966.0, 'coal': 390.0, 'gas': 2906.0, 'oil': 169.0, 'wind': 4622.0, 'biomass': 777.0, 'solar': 3354.0, 'hydro': 4867.0}, 'storage': {'hydro': -691.0}, 'source': 'rte-france.com'}

with new:

{'zoneKey': 'FR', 'source': 'https://opendata.reseaux-energies.fr', 'production': {'oil': 169.0, 'nuclear': 37966.0, 'gas': 2906.0, 'biomass': 777.0, 'coal': 390.0, 'solar': 3354.0, 'wind': 4622.0, 'hydro': 4867.0}, 'storage': {'hydro': -691.0}, 'datetime': datetime.datetime(2018, 8, 13, 15, 30, tzinfo=tzoffset(None, 7200))}

File: parsers/FR.py [hunk 1]
```diff
@@ -1,92 +1,85 @@
...
+        storage = {
+            'hydro': row[1]['pompage'] * -1 + row[1]['hydraulique_step_turbinage'] * -1
+        }
+
+        datapoints.append({
+            'zoneKey': zone_key,
+            'datetime': arrow.get(row[1]['date_heure']).datetime,
+            'production': production,
+            'storage': storage,
+            'source': 'https://opendata.reseaux-energies.fr'
```
Comment (corradio) [inline parsers/FR.py]: we normally only put the domain here. Can you remove "https://" ?

Comment (lorrieq) [reply parsers/FR.py]: Sure, will I also update the README?

Comment (corradio) [reply parsers/FR.py]: Ah yes that's probably a good idea.

**Comment by lorrieq** [comment]
Actually do you want to create an API key for the electricitymap and I'll change the code to read it from `os.environ['RESEAUX_ENERGIES_TOKEN']`?

**Comment by corradio** [comment]
That's probably a good idea.

**Comment by lorrieq** [comment]
Think that's everything!

**Pull request merged**